### PR TITLE
Moving navigation to JS for LEx and SF1

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes
 - Refreshing page in Lightning Experience after contact deletion.

# Issues Closed

### CODE REVIEW

**Comment by jjbennett** [comment]
@njjc this is ready, just a note though that I am grabbing a param in LEx that is undocumented but is available in LEx and SF1.

File: src/classes/CON_DeleteContactOverride_CTRL.cls [hunk 1]
```diff
@@ -63,8 +66,11 @@ public with sharing class CON_DeleteContactOverride_CTRL {
     * account should be deleted.
     */ 
     public CON_DeleteContactOverride_CTRL(ApexPages.StandardController stdController) {
-        retURL = ApexPages.currentPage().getParameters().get('returl');
-        
+        sforceUser = UserInfo.getUiThemeDisplayed().contains('Theme4');
```
Comment (njjc) [inline src/classes/CON_DeleteContactOverride_CTRL.cls]: Is this documented anywhere?

Comment (jjbennett) [inline src/classes/CON_DeleteContactOverride_CTRL.cls]: Yes it is here: https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_methods_system_userinfo.htm

**Comment by jjbennett** [comment]
@njjc added the items we spoke about this morning.

**Comment by njjc** [comment]
Awesome work @jjbennett, thank you!

**Pull request merged**